microsoft/PowerToys
# Import the source code for Window Walker

Import the codebase for [Window Walker](https://github.com/betsegaw/windowwalker/) without loading it into the module list nor adding it to the installer list.

## Summary of the Pull Request

A couple of things to note,
- You won't be able to try out window walker in PowerToys until you add WindowWalker back into the module list (reverting the change [here](https://github.com/microsoft/PowerToys/commit/e958414e1cebff072ac8a293d7e5e1fa67b770b1))
- The code is essentially the same as what is currently sitting on the develop branch in [betsegaw/windowwalker](https://github.com/betsegaw/windowwalker/tree/develop) except that Window Walker won't show on first launch to be consistent with the other PowerToys.

 
## References
#44 - Win+R replacement

## PR Checklist
* [X] Applies to #861 alt-tab quick selector
* ~~Tests added/passed~~
* ~~Requires documentation to be updated~~


## Validation Steps Performed
- Tested the when activated Window Walker launches and the other Power Toys launch as well. Currently, this module is just an import of the existing Window Walker code so the goal was just to make sure not to break PowerToys.
